server/cmd/mmctl/docs/mmctl_import_process.rst

```diff
@@ -27,7 +27,8 @@ Options
 
 ::
 
-  -h, --help   help for process
+  -h, --help     help for process
+      --legacy   Set this to indicate that the file path passed exists on the server side.
```

Comment:
0/5 on wording but I think we need to be clear that the file was uploaded with `mmctl import upload` here too